pyiron_base/_tests.py

```diff
@@ -10,6 +10,7 @@
 import unittest
 from os.path import split, join
 from os import remove
+import posixpath
```

Comment:
Just for consistency is there a reason why `os.path.join()` is not sufficient?

Reply:
> 
> 
> Just for consistency is there a reason why `os.path.join()` is not sufficient?

The `posixpath.join` method is [platform agnostic](https://www.georgevreilly.com/blog/2016/07/12/PythonJoiningUrlsWithPosixpathJoin.html) which is important esp for windows.

Reply:
I guess `posixpath.join` is platform independent ([source](https://www.georgevreilly.com/blog/2016/07/12/PythonJoiningUrlsWithPosixpathJoin.html)) so we should use this.

Reply:
`os.path.join` should be doing the correct™ thing depending on the platform.

Reply:
> I guess `posixpath.join` is platform independent ([source](https://www.georgevreilly.com/blog/2016/07/12/PythonJoiningUrlsWithPosixpathJoin.html)) so we should use this.

The link talks about URL not file paths. `posixpath` always gives you `/` while `os.path` gives `/` on linux and Mac OS X and `\` on windows. We want to use `posixpath` for all HDF5 internal path but for file path I would suggest to use `os.path.join` to be more consistent with the python standards.